_docs/company/partnerships.md

```diff
@@ -12,23 +13,23 @@ We're always looking for innovative ways to improve and [spread DuckDuckGo](http
 
-*   Please don't use DuckDuckGo in the title. (There can be exceptions so please contact us if you think you have a case for doing so). Mentioning us in the description is fine.
-*   Please don't use Dax (our mascot).
-*   Please don't pair our search with something that has another privacy policy, like another auto-complete service that tracks people, as doing so gives users a false sense of privacy.
-*   Please don't include our search results in any sort of frame (whether desktop or mobile) that makes our site embedded within another Web application with the notable exception of browser integration. We simply don't have the rights to allow framing based on how [we generate our search results](/results/sources). For clarity, browser integration where there is an address bar and our site (with branding) takes up the full screen is just fine and encouraged!
-*   Please don't use our [URL parameters](https://duckduckgo.com/params) to strip our branding from our site.
-*   Please give us attribution where it makes sense, and where you do link back to the [encrypted version of our homepage](https://duckduckgo.com/).
+-   Please don't use DuckDuckGo in the title. (There can be exceptions so please contact us if you think you have a case for doing so). Mentioning us in the description is fine.
+-   Please don't use Dax (our mascot).
+-   Please don't pair our search with something that has another privacy policy, like another auto-complete service that tracks people, as doing so gives users a false sense of privacy.
+-   Please don't include our search results in any sort of frame (whether desktop or mobile) that makes our site embedded within another Web application with the notable exception of browser integration. We simply don't have the rights to allow framing based on how [we generate our search results](/results/sources). For clarity, browser integration where there is an address bar and our site (with branding) takes up the full screen is just fine and encouraged!
+-   Please don't use our [URL parameters](https://duckduckgo.com/params) to strip our branding from our site.
+-   Please give us attribution where it makes sense, and where you do link back to the [encrypted version of our homepage](https://duckduckgo.com/).
 
 If you have any questions about applying these guidelines in your specific situation, please [get in touch.](mailto:open@duckduckgo.com)
 
 ### DuckDuckGo as a Search Option
 
 We have many partnerships with apps, browsers and distributions that include us as a search option:
 
-*   [Browsers](/desktop/other-browsers)
-*   [Linux distributions](/desktop/linux-distributions)
-*   [iOS](/mobile/ios)
-*   [Android](/mobile/android)
+-   [Browsers](/desktop/other-browsers)
+-   [Linux distributions](/desktop/linux-distributions)
+-   [iOS](/mobile/ios)
+-   [Android](/mobile/android)
```

Comment:
These (and other similar root-absolute links) do not work on the demo. As a part of your overall linting (which is very appreciated!), could we add `{{ site.baseurl }}` to links across the board to ensure that they work no matter where they're deployed?

Reply:
This will involve changing all the markdown styles links to <a> links since using variables within markdown links doesn't work.

I agree it should be done, but not as a blocker on shipping this PR.